subiquity/common/filesystem/tests/test_sizes.py

```diff
@@ -109,6 +113,17 @@ def test_weighted_split(self):
                 minimum=50 << 30, recommended=200 << 30, maximum=230 << 30)
         self.assertEqual(expected, actual)
 
+    def test_unaligned_size(self):
+        for i in range(100):
+            actual = calculate_guided_resize(
+                part_min=40 << 30,
+                part_size=(240 << 30) + random.randrange(MiB),
```

Comment:
When a test has a random element, I like to see that value be logged.  If it ever fails that value will be critical.
```
diff --git a/subiquity/common/filesystem/tests/test_sizes.py b/subiquity/common/filesystem/tests/test_sizes.py
index b1a7d9b8..2d96bfb4 100644
--- a/subiquity/common/filesystem/tests/test_sizes.py
+++ b/subiquity/common/filesystem/tests/test_sizes.py
@@ -115,11 +115,12 @@ class TestCalculateGuidedResize(unittest.TestCase):
 
     def test_unaligned_size(self):
         for i in range(100):
+            rr = random.randrange(MiB)
             actual = calculate_guided_resize(
                 part_min=40 << 30,
-                part_size=(240 << 30) + random.randrange(MiB),
+                part_size=(240 << 30) + rr,
                 install_min=10 << 30)
-            self.assertEqual(actual.install_max % MiB, 0)
+            self.assertEqual(actual.install_max % MiB, 0, rr)
             self.assertEqual(actual.minimum % MiB, 0)
             self.assertEqual(actual.recommended % MiB, 0)
             self.assertEqual(actual.maximum % MiB, 0)
```